provisioning/tasks/init-debian.yml

```diff
@@ -20,19 +20,24 @@
   apt_repository: repo='ppa:ondrej/apache2'
   when: ansible_distribution_release == "precise" and ansible_distribution == "Ubuntu"
 
-- name: Add repository for PHP 5.5.
-  apt_repository: repo='ppa:ondrej/php5'
-  when: php_version == "5.5" and ansible_distribution == "Ubuntu" and ansible_distribution_version == "12.04"
+- name: Add repository for PHP on Ubuntu.
+  apt_repository: repo='ppa:ondrej/php'
+  when: ansible_distribution == "Ubuntu" and ansible_distribution_version != "16.04"
```

Comment:
@thom8 / @oxyc - FYI, note that as of https://github.com/geerlingguy/ansible-role-php/pull/135 (latest PHP role version), you can set `php_packages_state: "latest"`, and that, along with a different repo (yum) or PPA (apt), will switch PHP versions on the fly as well.

I just used that to finally bump hostedapachesolr.com to a modern PHP version on CentOS 6 (after testing on staging servers of course) :D